guide/deployment_digitalocean.md

```diff
@@ -0,0 +1,134 @@
+```
+docker-machine ssh node-2 -- docker swarm join --token SWMTKN-1-239v0epdnhuol2ldguttncoaleovy29hnwyglde0kba1owc9ng-9488z5we2invwcn69f5flq7uu 104.131.69.233:2377
+```
+Repeat for `node-3`.
+```
+docker-machine ssh node-3 -- docker swarm join --token SWMTKN-1-239v0epdnhuol2ldguttncoaleovy29hnwyglde0kba1owc9ng-9488z5we2invwcn69f5flq7uu 104.131.69.233:2377
+```
+
+## Configure Docker CLI to use remote Swarm
+Run this command each time you open a new shell, this tells Docker where your remote Swarm is.
+```
+eval $(docker-machine env node-1)
+```
+
+## Deploy the OpenFaaS Stack
+```
+$ git clone https://github.com/alexellis/faas && \
+  cd faas && \
+  git checkout 0.6.5 && \
+  ./deploy_stack.sh
+```
+
+## Test the UI
+Within a few seconds (or minutes if on a poor WiFi connection) the API gateway and sample functions will be deployed to the Swarm cluster running on DigitalOcean.
+
+Access the Gateway UI via the IP address returned by `docker-machine ip node-1` (you can also access via `node-2` and `node-3`):
+```
+echo http://$(docker-machine ip node-1):8080
+```
+Prometheus metrics can be viewed at:
+```
+echo http://$(docker-machine ip node-1):9090
+```
+
+## Deleting OpenFaaS Droplets
+You can use `docker-machine` to delete any created Droplets if are finished with your OpenFaaS deployment.
+```
+docker-machine rm node-1 node-2 node-3
+```
+
+## Creating a Load Balancer
+Rather than address each node individually it can be preferrable to run behind a load balancer.
+
+From the DigitalOcean console Networking page, open the Load Balancers tab and click *Create Load Balancer*.
+
+Give the balancer a name and select the Droplets which will be balanced using the `openfaas-getting-started` tag and `NYC3` region (these were values passed to docker-machine when creating the nodes).
+
+![create_lb](https://user-images.githubusercontent.com/83862/30240233-274c4dc0-9564-11e7-8881-54bce652392f.jpg)
+
+Update the forwarding rules to point at the Gateway on `8080` and Prometheus dashboard on `9090`:
+![forwarding_rules](https://user-images.githubusercontent.com/83862/30240106-0eb71242-9562-11e7-846e-093627026a7c.jpg)
+
+Expand the Advanced section and update the health check to use port `8080`.
+![health_checks](https://user-images.githubusercontent.com/83862/30240104-0e98e3d0-9562-11e7-89b6-c266384e35d8.jpg)
+
+Click `Create Load Balancer` and after a few minutes your balancer will be available.
+
+![balancer_ready](https://user-images.githubusercontent.com/83862/30240232-2747becc-9564-11e7-867a-c3ac220f2ae3.png)
+
+You can now access the OpenFaaS Gateway, Prometheus dashboard and all functions via the load balanced IP address. For example from the balancer above:
+- Gateway: http://45.55.124.29:8080
+- Prometheus: http://45.55.124.29:9090
+
+## Costs
+
+The costs of running your own OpenFaaS platform on DigitalOcean depend on how resiliant you want it to be, from the simplest single node deployment, to a three node swarm behind a load balancer.
+
```

Comment:
This is interesting but I'm not sure about including this info - since it will change and we have to keep it updated.

Reply:
Yep, good point

Reply:
Could be a good blog post or Tweet - especially if qualified with a date.

Reply:
ok, I'll push the current contents out as a blog post and refactor to remove redundancy across docs.